app/controllers/suggested_councillors_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SuggestedCouncillorsController < ApplicationController
+  def new
+    @suggested_councillor = SuggestedCouncillor.new
+    @authority = Authority.find(params[:authority_id])
+  end
+
+  def create
+    @suggested_councillor = SuggestedCouncillor.new(suggested_councillor_params)
+    @authority = Authority.find(params[:authority_id])
+    @suggested_councillor.authority= @authority
```

Comment:
Would it be possible to assign this when the SuggestedCouncillor is initialized if `@authority` is assigned first?